lib/rules/length-zero-no-unit/index.mjs

```diff
@@ -49,14 +52,13 @@ const rule = (primary, secondaryOptions, context) => {
 
 		if (!validOptions) return;
 
-		let needsFix;
-
 		/**
-		 * @param {import('postcss').Node} node
+		 * @param {AtRule | Declaration} node
 		 * @param {number} nodeIndex
-		 * @param {import('postcss-value-parser').Node} valueNode
+		 * @param {Node} valueNode
+		 * @param {ParsedValue} parsedValue
 		 */
-		function check(node, nodeIndex, valueNode) {
+		function check(node, nodeIndex, valueNode, parsedValue) {
```

Comment:
I don't like having functions with more than 2 arguments but it seemed like the most straightforward refactor.
I am trying to limit the diff to the minimum.